.github/workflows/node-js-ci.yml

```diff
@@ -13,10 +13,10 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm ci
-      - run: npm ci
+      - run: npm i
```

Comment:
The modern package-lock.json format broke `npm ci`. I've removed package-lock.json (it's pointless for modules anyway) and changed to `npm i` since `npm ci` only works with package-lock / shrinkwrap